### PR TITLE
one step for the big change, product, cart and order need to be suite…

### PR DESCRIPTION
**Pull request by eilonasher**
… for the last version we talked about:

product has only department, smell, maybe description - need to talk about it, price per liter
the other departments will be changed in the fit branches

### CODE REVIEW

File: app/src/main/java/com/example/sabonit/Product.java [hunk 1]
```diff
@@ -20,9 +20,7 @@
     // Price of one liter of the product, in use when computing the price of a given amount.
     private int pricePerLiter;
     // Optional, a short description or anything that not reflected in the product's department and name.
-    private String description;
-//
-    private String imageUrl;
+    private String description; //todo last decision
```
Comment (liormoalem) [inline app/src/main/java/com/example/sabonit/Product.java:23]: we need to talk about it

File: app/src/main/java/com/example/sabonit/Product.java [hunk 2]
```diff
@@ -44,11 +42,10 @@ public Product() {
      * @param smell - a specific product smell in the department, probably the manufacturer name.
      * @param pricePerLiter - price of one liter of the product.
      */
-    public Product(String department, String smell, int pricePerLiter, String imageUrl) {
+    public Product(String department, String smell, int pricePerLiter) {
```
Comment (liormoalem) [inline app/src/main/java/com/example/sabonit/Product.java:45]: agrreed

**Review by liormoalem — COMMENTED** [review]
only need to talk about descriptin, can merge it

**Review by yairelboher — COMMENTED** [review]
Agree, you can merge it :)

**Pull request merged**